2-ui/1-document/01-browser-environment/article.md

```diff
@@ -1,113 +1,113 @@
-# Browser environment, specs
+# ব্রাউজার এনভায়রনমেন্ট, স্পেসিপিকেশন
 
-The JavaScript language was initially created for web browsers. Since then it has evolved and become a language with many uses and platforms.
+শুরুতে জাভাস্ক্রিপ্ট তৈরি করা হয়েছিল ওয়েব ব্রাউজারের জন্য। তবে বর্তমানে এটি বিভিন্ন প্লাটফর্মে ব্যবহার করা হচ্ছে।
 
-A platform may be a browser, or a web-server or another *host*, even a "smart" coffee machine, if it can run JavaScript. Each of them provides platform-specific functionality. The JavaScript specification calls that a *host environment*.
+যদি জাভাস্ক্রিপ্ট রান করা যায় তাহলে প্লাটফর্মসমূহ  হতে পারে ব্রাউজার, সার্ভার, অথবা অন্য কোন *হোস্ট*, এমনকি স্মার্ট কফি মেশিনও। প্রতি ক্ষেত্রে নিজস্ব প্লাটফর্ম অনুযায়ী নিজস্ব কিছু ফাংশনালিটি থাকে। জাভাস্ক্রিপ্টের স্পেসিপিকেশন অনুযায়ী একে বলা হয় *হোস্ট এনভায়রনমেন্ট*।
```

Comment:
Please rephrase this sentence:

`যদি জাভাস্ক্রিপ্ট রান করা যায় তাহলে প্লাটফর্মসমূহ হতে পারে ব্রাউজার, সার্ভার, অথবা অন্য কোন *হোস্ট*, এমনকি স্মার্ট কফি মেশিনও। প্রতি ক্ষেত্রে নিজস্ব প্লাটফর্ম অনুযায়ী নিজস্ব কিছু ফাংশনালিটি থাকে। জাভাস্ক্রিপ্টের স্পেসিপিকেশন অনুযায়ী একে বলা হয় *হোস্ট এনভায়রনমেন্ট*।`

likewise:

`বর্তমানে প্রায় সকল প্লাটফর্মে জাভাস্ক্রিপ্ট রান করা যায়, সেটা হতে পারে ব্রাউজার, সার্ভার, অথবা অন্য কোন *হোস্ট*, এমনকি স্মার্ট কফি মেশিনেও। প্রতি ক্ষেত্রে নিজস্ব প্লাটফর্ম অনুযায়ী নিজস্ব কিছু ফাংশনালিটি থাকে। জাভাস্ক্রিপ্টের স্পেসিফিকেশন অনুযায়ী একে বলা হয় *হোস্ট এনভায়রনমেন্ট*।`